### PR TITLE
Update Directions and Content of CONTRIBUTING.md

### LINKED ISSUE #1124: Update Directions and Content of CONTRIBUTING.md
**Is your feature request related to a problem? Please describe.**
- It is not made clear how to fork the project and submit a pull request in the CONTRIBUTING.md
- There are some inconsistencies with the project's documentation [webpage](https://spotipy.readthedocs.io/en/2.23.0_a/#contribute) (the authors/contributors section is not included)

**Describe the solution you'd like**
- Include a step at the beginning that explains to newer programmers how to fork the repository
- Include a step at the end that explains how to submit a pull request
- Include a section at the end that displays the authors/contributors of the project

**Describe alternatives you've considered**
N/A

**Additional context**
[CONTRIBUTING.md](https://github.com/spotipy-dev/spotipy/files/15487178/CONTRIBUTING.md)
[CHANGELOG.md](https://github.com/spotipy-dev/spotipy/files/15487180/CHANGELOG.md)

### PR DESCRIPTION
**Pull request by bennetthamilton**
Resolves #1124

I contributed to the project’s documentation by updating the [CONTRIBUTING.md](https://github.com/spotipy-dev/spotipy/blob/master/CONTRIBUTING.md) document. The ‘contributing.md’ is intended to give instructions to developers on how to make changes. However, it does not give very detailed instructions on how to fork and make pull requests. Existing instructions are not very clear for beginner developers. 

In addition, there is an inconsistency between their repo and main [website](https://spotipy.readthedocs.io/en/2.24.0/#contribute). The main website has a list of authors/contributors under the “Contribute” section which is missing in corresponding document.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,3 +104,29 @@ Don't forget to add a short description of your change in the [CHANGELOG](CHANGE
...
+Ryan Choi (ryankicks on Github)
+Simon Metson (drsm79 on Github)
+Steve Winton (swinton on Github)
+Tim Balzer (timbalzer on Github)
+corycorycory on Github
+Nathan Coleman (nathancoleman on Github)
+Michael Birtwell (mbirtwell on Github)
+Harrison Hayes (Harrison97 on Github)
+Stephane Bruckert (stephanebruckert on Github)
+Ritiek Malhotra (ritiek on Github)
```
Comment (dieser-niko) [inline CONTRIBUTING.md:132]: As much as I appreciate every contributor, I don't think we need to plaster their names all over the place.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -48,16 +60,40 @@ To make sure if the import lists are stored correctly:
...
 
  - Bump version in setup.py
  - Bump and date changelog
  - Add to changelog:
 
        ## Unreleased
-       Add your changes below.
 
        ### Added
+       - Replace with changes
```
Comment (dieser-niko) [inline CONTRIBUTING.md:96]: If you're going to remove the line above, you could add the `- Replace with changes` to Fixed and Removed as well.

**Review by dieser-niko — CHANGES_REQUESTED** [review]
The rest looks good

File: CONTRIBUTING.md (hunk 1 above)
Comment (stephanebruckert) [reply CONTRIBUTING.md:132]: True! We have https://github.com/spotipy-dev/spotipy/graphs/contributors for that

Also this list I outdated, I guess you've taken it from https://github.com/spotipy-dev/spotipy/blob/master/docs/index.rst#contribute - any chance you can remove that list from there too? We can just keep the technical instructions

**Review by stephanebruckert — CHANGES_REQUESTED** [review]
I don't think any of these information are needed, as they are extremely basic github usage instructions: forking/cloning/pushing/prs. We shouldn't have to add that to every repo on Github.

If you want to keep your PR, please only remove the contributors from https://github.com/spotipy-dev/spotipy/blob/master/docs/index.rst#contribute, otherwise we can close it.

**Comment by stephanebruckert** [comment]
Fixed in https://github.com/spotipy-dev/spotipy/commit/ef282e2423d3ffaff19119e5f36636b29724e86e

**Pull request closed without merging**